Migrate Cloudcraft documentation from Help Scout to Datadog (#21358)
DataDog/documentation: content/en/cloudcraft/faq/delete-account.md

---
title: How do I delete my account?

> kind: guide

Reviewer: ```suggestion
kind: faq
```

---

> You can delete your Cloudcraft account from inside the Cloudcraft application or by contacting support for paid users.
> 
> If you are absolutely certain that you want to proceed, remember that there is no turning back! After an account is deleted, we don't keep any data on our servers, so make sure to export your data if you have to.

Reviewer: ```suggestion
You can delete your Cloudcraft account from the Cloudcraft application, or by contacting support. 

When an account is deleted, Cloudcraft removes the account's data from its servers. Be sure to export a copy of your data before deleting your account.
```

> ## Delete your free Cloudcraft account
> 
> 1. Click the **User** icon in the top-right hand corner of Cloudcraft and select the **User settings** option from the dropdown menu.
> 2. Click the **Delete account** button at the bottom of the page.
> 3. Confirm you want to _immediately_ delete your account by clicking the red **Delete** button.
> 
> ## Delete your paid Cloudcraft account
> 
> The process of deleting a paid account is currently only possible from our side, and it takes effect immediately.
> 
> If that is what you want, please [get in touch with our support team from inside Cloudcraft][1], and they will handle this for you.

Reviewer: ```suggestion
## Delete a free Cloudcraft account

1. Click the **User** icon in the upper-right corner of Cloudcraft and select **User settings** from the dropdown menu.
2. Click **Delete account** at the bottom of the page.
3. Confirm you want to _immediately_ delete your account by clicking the red **Delete** button.

## Delete a paid Cloudcraft account

To delete a paid account, contact [the support team][1].
```
